connectors/src/api/webhooks/webhook_intercom.ts

```diff
@@ -101,8 +102,18 @@ const _webhookIntercomAPIHandler = async (
     return res.status(200).end();
   }
 
-  // Check we have the permissions to sync this conversation
+  if (connector?.pausedAt) {
```

Comment:
Given that we check on line 95.
```suggestion
  if (connector.pausedAt) {
```